src/test/resources/application-test.yaml

```diff
@@ -0,0 +1,9 @@
+spring:
+  datasource:
+    url: ${TEST_URL}
+    username: ${TEST_USERNAME}
+    password: ${TEST_PASSWORD}
+    driver-class-name: org.testcontainers.jdbc.ContainerDatabaseDriver
+  jpa:
+    hibernate:
+      ddl-auto: create
```

Comment:
@cichan02 we are not using any ORM here

we manage our schema using Liquibase instead